cdap-docs/admin-manual/source/installation/installation.rst

```diff
@@ -127,10 +127,10 @@ Once you have installed the JDK, you'll need to set the JAVA_HOME environment va
 
 Node.js Runtime
 +++++++++++++++
-You can download the latest version of Node.js from `nodejs.org <http://nodejs.org>`__:
+You can download the appropriate version of Node.js (from v0.8.16 through v0.10.32) from `nodejs.org <http://nodejs.org>`__:
 
-1. Download the appropriate Linux or Solaris binary ``.tar.gz`` from
-   `nodejs.org/download/ <http://nodejs.org/download/>`__.
+#. Download the appropriate Linux or Solaris binary ``.tar.gz`` for a version in the above range from
+    `nodejs.org/dist/ <http://nodejs.org/dist/>`__.
```

Comment:
remove the leading space (just before the \` ) so that the \`  lines up with the "D"